templates/consul.json

```diff
@@ -0,0 +1,52 @@
+[
+        {
+            "name": "consul_agent-${env}",
+            "image": "${image}",
+            "memoryReservation": 512,
+            "environment": [
+                {
+                    "name": "CONSUL_BIND_INTERFACE",
+                    "value": "eth0"
+                }
+            ],
+            "command": [
+              "agent", "-client=0.0.0.0"
+            ],
+            "mountPoints": [
+              {
+                "sourceVolume": "consul-config-dir",
+                "containerPath": "/consul/config"
+              }
+            ],
+            "logConfiguration": {
+              "logDriver": "awslogs",
+                "options": {
+                  "awslogs-group": "${awslogs_group}",
+                   "awslogs-region": "${awslogs_region}",
+                   "awslogs-stream-prefix": "${awslogs_stream_prefix}"
+                }
+              }
+        },
+        {
+            "name": "registrator-${env}",
+            "image": "gliderlabs/registrator",
```

Comment:
let's either pin to a specific release of registrator, or alternately specify `gliderlabs/registrator:latest` to make it clear that we're not pinning